src/main/java/org/csanchez/jenkins/plugins/kubernetes/PodTemplate.java

```diff
@@ -969,6 +990,7 @@ public YamlMergeStrategy getDefaultYamlMergeStrategy() {
     @Override
     public String toString() {
         return "PodTemplate{" +
+                (id == null ? "" : "inheritFrom='" + inheritFrom + '\'') +
```

Comment:
Should this be `(id == null ? "" : "id='" + id + '\'')`

Reply:
yikes, copy-pasta, forgot to update.